triage/models/dsapp_scaler.org

```diff
@@ -0,0 +1,570 @@
+* DsappCutOff
```

Comment:
What is this file? It seems like it has useful content but i don't really know how it fits in overall.